scalableminds/webknossos
# Rename oxalis to webknossos in config files

Steps to test:
- See jenkins tests
- `docker-compose up webknossos`

Issues:
- fixes #1602

------


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1612/create?referer=github" target="_blank">Log Time</a>
